### PR TITLE
Morse decoding

### PR DESCRIPTION
**Pull request by eamador1**
In this project, the following was done:
---

- create a method to decode a Morse code character ✅
- create a method to decode an entire word in Morse code ✅
- create a method to decode the entire message in Morse code ✅

### CODE REVIEW

File: morse_decode.rb
Comment (aamirkhan2478) [inline morse_decode.rb:1]: - [ ] Please follow the requirement in the requirement you will create 3 methods but in your case, you just created one method so kindly follow the below instructions to implement the changes :slightly_smiling_face: 

- Create a method for decoding single uppercase characters eg: `decide_char(".-")` returns `"A"`
- Create a method for decoding the entire word and every character in a word will be separated by a single space (e.g. `decode_word("-- -.--")` returns `"MY"`).
- Create a method for decoding the entire message in Morse code and Every word will be separated by 3 spaces (e.g. `decode("-- -.--   -. .- -- .")` returns `"MY NAME"`).
- Reuse methods whenever possible. Make sure that your code is DRY.
- Once your code is ready use it to decode the message from the old bottle (`      .-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...`)

**Review by aamirkhan2478 — CHANGES_REQUESTED** [review]
Hi @eamador1 & @Estete9 , 



Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


## Highlights
✔️ Good Effort

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by eamador1** [comment]
Since the last code review, the following was done
---

- Create a method for decoding single uppercase characters eg: decide_char(".-") returns "A"
- Create a method for decoding the entire word and every character in a word will be separated by a single space (e.g. decode_word("-- -.--") returns "MY").
- Create a method for decoding the entire message in Morse code and Every word will be separated by 3 spaces (e.g. decode("-- -.--   -. .- -- .") returns "MY NAME").
- Reuse methods whenever possible 
- Make sure that your code is DRY.
- Once the code is ready use it to decode the message from the old bottle (      .-   -... --- -..-   ..-. ..- .-.. .-..   --- ..-.   .-. ..- -... .. . ...)

**Review by skyv26 — APPROVED** [review]
## Hi @eamador1 and @Estete9, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit:  
Congratulations! 🎉 

## STATUS: APPROVED 🟢 



## Highlights
- Implemented the features as per the previous code reviewer's request ✅ 

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._

## ⚠️  WARNING ⚠️ 
**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

## Have any doubt ❓ 
Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag @skyv26 in your question so I can receive the notification. You can also follow me on the below platforms

<table>
<tr>
<td>

<a href="https://github.com/skyv26">
   @skyv26
</a>

</td>
<td>

<a href="https://www.linkedin.com/in/skyv2022/">
   linkedin-skyv2022
</a>

</td>
<td>

<a href="https://twitter.com/vrma_aakash">
   @vrma_aakash
</a>

</td>
<td>

<a href="https://www.instagram.com/skybrel/">
   @skybrel
</a>

</td>
<td>

<a href="https://www.frontendmentor.io/profile/skyv26">
   @skybrel
</a>

</td>
</tr>
</table>

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**